app/src/main/java/org/dhis2/utils/WebViewActivity.kt

```diff
@@ -9,9 +10,8 @@ import android.webkit.WebViewClient
 import androidx.databinding.DataBindingUtil
 import org.dhis2.R
 import org.dhis2.databinding.ActivityWebviewBinding
-import org.dhis2.usescases.general.ActivityGlobalAbstract
 
-class WebViewActivity : ActivityGlobalAbstract() {
+class WebViewActivity : Activity() {
```

Comment:
Which was the problem that was generating `ActivityGlobalAbstract`?

Reply:
I believe ActivityGlobalAbstract access the databaset @andresmr